### PR TITLE
#297 - TL Header has a suppress-call-to-action input attribute

### LINKED ISSUE #297: The TL Header should support a `suppress-call-to-action` input attribute.
This will require the addition on a new `@Input()` called `suppressCallToAction` when `'true'` this should be used with an `*ngIf` to hide the giving button.

Any styles and classes which reference the `supportTheLibraries` should be refactored to  `call-to-action`

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #297 - TL Header has a suppress-call-to-action input attribute

### CODE REVIEW

File: src/app/tl-header/tl-header.component.scss [hunk 1]
```diff
@@ -126,7 +126,7 @@
 
       .information-for {
         a {
-          color: var(--secondary-button-hover-bg); 
+          color: var(--secondary-button-hover-bg);
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.scss:129]: Should this be the `secondary-default-hover-bg`

File: src/app/tl-header/tl-header.component.html [hunk 2]
```diff
@@ -283,7 +283,7 @@
 
 <ng-template #bottomNavContent>
     <ng-content select="wvre-nav-li"></ng-content>
-    <wvr-nav-li-component class="ml-auto supportTheLibraries">
+    <wvr-nav-li-component class="ml-auto call-to-action" *ngIf="!suppressCallToAction">
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.html:286]: this should eveulate by  a string, `suppressCallToAction==='true'`

File: src/app/tl-header/tl-header.component.ts [hunk 3]
```diff
@@ -44,6 +44,9 @@ export class TamuHeaderComponent extends TamuAbstractBaseComponent {
 
   @Input() pageTitleUrl = '/';
 
+  /** This boolean attribute is used to supress display of "Give to the Libraries" button. */
+  @Input() suppressCallToAction: 'true' | 'false' = 'true';
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.ts:48]: this should default to 'false'

**Review by jeremythuff — CHANGES_REQUESTED** [review]

File: src/app/tl-header/tl-header.component.ts (hunk 3 above)
Comment (rmathew1011) [reply src/app/tl-header/tl-header.component.ts:48]: Resolved this.

**Review by jeremythuff — APPROVED** [review]

**Pull request merged**